c++/src/kj/compat/tls.c++

```diff
@@ -489,6 +490,13 @@ TlsContext::Options::Options()
 // Classic DH is arguably obsolete and will only become more so as time passes, so perhaps we'll
 // never bother.
 
+struct TlsContext::SniCallback {
+  // struct SniCallback exists only so that callback() can be declaned in the .c++ file, since it
```

Comment:
s/declaned/declared/

Reply:
Whoa, what brings you here? :)

(fixed)

Reply:
I'm still a watcher on capnproto and sandstorm-io, so I get emails from GitHub for each PR.  Don't always find the time to look over them, and I rarely have anything to contribute, but when the opportunity arises... :)